plugin/trino-ranger/README.md

```diff
@@ -0,0 +1,95 @@
+# Trino-Ranger plugin
+
+This plugin is designed to be build and run inside Trino.
+
+It works with vanilla Apache Ranger, version 2.2.1 and up. You dont need to customize Ranger at all.
+
+Here are the setup steps.
+
```

Comment:
```suggestion
Apache Ranger is a framework to enable comprehensive data security across many platforms, including Trino. Rather than maintain the plugin with the Apache Ranger project, this plugin is designed to be built and run inside Trino.

It works with vanilla Apache Ranger, version 2.2.1 and up. You don't need to customize Ranger at all.
```

Reply:
Apache Ranger is trademarked by the ASF, so saying vanilla Apache Ranger seems a bit redundant?

Reply:
@cbaenziger vanilla Ranger means ZERO changes in Ranger. Works out of the box. Happy to change the wording.